eng/common/mcp/azure-sdk-mcp.ps1

```diff
@@ -8,23 +11,61 @@ param(
     [string]$Repository = 'Azure/azure-sdk-tools',
     [string]$RunDirectory = (Resolve-Path (Join-Path $PSScriptRoot .. .. ..)),
     [switch]$Run,
-    [switch]$UpdateVsCodeConfig,
-    [switch]$Clean
+    [switch]$UpdateVsCodeConfig
 )
 
 $ErrorActionPreference = "Stop"
-
-if (-not $InstallDirectory)
-{
-    $homeDir = if ($env:HOME) { $env:HOME } else { $env:USERPROFILE }
-    $InstallDirectory = (Join-Path $homeDir ".azure-sdk-mcp" "azsdk")
-}
 . (Join-Path $PSScriptRoot '..' 'scripts' 'Helpers' 'AzSdkTool-Helpers.ps1')
 
-if ($Clean) {
-    Clear-Directory -Path $InstallDirectory
+function getInstallDirectory {
```

Comment:
```suggestion
function Get-InstallDirectory {
```

Reply:
I named it this way to kind of mark it as private vs. the powershell external cmdlet convention.

Reply:
but actually this could be useful elsewhere. Maybe I should move this to common